From 96d63cfdfdbe59566f988ede5182cc949cba1c72 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Tue, 31 Oct 2017 11:10:31 +0800
Subject: [PATCH] ARM: rockchip_defconfig: add CONFIG_PM_ADVANCED_DEBUG config

open debug node cat d/pm_genpd/pm_genpd_summary for pd summary

Change-Id: I6ad1c94ea8c5eb61a99235602825b19533c098f4
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 17ee265bcba7..b84d0b4a8f71 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -79,6 +79,7 @@ CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
 CONFIG_PM_DEBUG=y
+CONFIG_PM_ADVANCED_DEBUG=y
 CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_UNIX=y
-- 
2.35.3

